tests/test_models.py

```diff
@@ -0,0 +1,90 @@
+from .base import BaseTinyFatTest
+from tinyfatdb import TinyFatDB, TinyFatTable
+from tinydb import Query as Q
+
+
+########################################################################
+class TestFatModels(BaseTinyFatTest):
+    """
+    Tests to ensure custom methods on instances of TinyFatModel work,
+    both for the default table and for additional tables.
+    """
+
+    ####################################################################
+    def setUp(self):
+        super(TestFatModels, self).setUp()
+        self.db = TinyFatDB.create(name="ABC", table=TinyFatTable)
+        self.table = self.db.table(name="Foo", table=TinyFatTable)
+        self.insertable = {"a": 1, "b": 2, "c": 3}
+        self.additional_insertable = {"short": "a", "medium": "aa", "long": "aaa"}
+
+    ####################################################################
+    def insert_entry(self, container, insertable):
+        """
+        Takes a container (db, or table) and returns a Model representation
+        of the entry that was inserted.
+        """
+        
+        eid = container.insert(insertable)
+        expected_entry = {**insertable, "eid": eid}
+        return expected_entry
```

Comment:
This function was necessary after I started using the TinyFatModel. The `ABCModel`/`FooModel` would not return the `eid` within dictionary. This was contrary to our natural behavior where we expect to have that key/value in the entry.